x-pack/solutions/security/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -19,7 +19,8 @@ export const policyFactory = (
   licenseUid = '',
   clusterUuid = '',
   clusterName = '',
-  serverless = false
+  serverless = false,
+  isGlobalTelemetryEnabled = false
 ): PolicyConfig => {
```

Comment:
Consider refactoring to an object.

Reply:
considered it, but there was a lot of usage, so skipped. 😁 but thanks to you, reconsidered and found that most of the usage doesn't pass any parameters, so it's easy to change : )
23faef95a90e69d3627185f25e2ba448b6650418

thanks for the suggestion!